src/microbe_stage/SpawnSystem.cs

```diff
@@ -147,6 +147,14 @@ private void SpawnEntities(Vector3 playerPosition, int existing)
 
         foreach (ISpawner spawnType in spawnTypes)
         {
+            // Spawining queued spawns
+            if (spawnType.QueuedSpawns.Count > 0)
+            {
+                Action spawn = spawnType.QueuedSpawns.Pop();
+                spawn();
+                spawned++;
+            }
```

Comment:
So this would be the entire spawn system global (the queued spawns) instead of per spawner, as this can still spawn a bunch of stuff per frame, and I don't want the extra fields to get added to the ISpawner definition.